references/detection/coco_utils.py

```diff
@@ -126,10 +126,6 @@ def _has_valid_annotation(anno):
             return True
         return False
 
-    if not isinstance(dataset, torchvision.datasets.CocoDetection):
```

Comment:
Instead of removing this (seemingly useless check) I could just add the same workaround as elsewhere i.e. add

```py
of isinstance(
        getattr(dataset, "_dataset", None), torchvision.datasets.CocoDetection
    ):
```